resources/ext.collections.styles/collections.less

```diff
@@ -90,4 +90,39 @@
 		}
 	}
 
+	.collection-card {
+		height: 14em;
```

Comment:
What do you think if we say `min-height: 6em`, like the `.collection-card-overlay` so that we don't show the white space until we get the images? Then when we have them we can restore this height